modules/test/playwright/package.json

```diff
@@ -30,7 +30,8 @@
 		"local:setup": "ts-node ./scripts/localSetup.ts",
 		"local:teardown": "ts-node ./scripts/localTeardown.ts",
 		"playwright": "playwright",
-		"test": "playwright test"
+		"test": "playwright test",
+		"testUI": "playwright test --ui"
```

Comment:
I'd use `test:ui`, as we do for `local:setup` and `local:teardown`... Or change those to 'localSetup` and `localTeardown`...  :shrug:

Reply:
Makes sense... I'll update it. Thanks.

Reply:
@izaera just updated with your suggestion.